### PR TITLE
Unknown tag value should not be reported as error.

### PR DESCRIPTION
**Pull request by pepper-ankan**
By the wording of [this Trello item](https://trello.com/c/gRuvkJ8f/24-change-opentsdb-to-not-report-an-error-on-an-unknown-tag-value), I'm interpreting this to mean "should not be _logged_ as errors", i.e., the _log message_ should not be an **ERROR**.  Hence, this CL only makes such changes.

I'm assuming that requests with a tag value that is unknown should still return an error status to the client, which seems to be a reasonable behavior; if this assumption is not correct, please let me know, and we can discuss this further.

Also, it would help to get an example of the kind of error message this is creating in a log.

### CODE REVIEW

**Comment by pepper-ankan** [comment]
Also, it would help to get an example of the kind of error message this is creating in a log.

**Comment by ssuchter** [comment]
I can't remember if we finalized on this being just about logging or the also the API response from TSD. I remember discussing both. Looking at this change assuming that the way you interpreted it is the intent - I do observe that the Dashboard does a good job of catching the TSD error and showing the response in a clear way.

File: src/tools/UidManager.java [hunk 1]
```diff
@@ -440,7 +440,7 @@ private static int rename(final HBaseClient client,
                 + " to " + newname, e);
       return 3;
     } catch (NoSuchUniqueName e) {
-      LOG.error(e.getMessage());
+      LOG.warn(e.getMessage());
```
Comment (ssuchter) [inline src/tools/UidManager.java]: I don't even thing it should be a warning. I think it should be an info. These fields come from the user (e.g. they typo their username) and we don't want WARNings in our logs based upon user error.

File: src/tools/UidManager.java [hunk 2]
```diff
@@ -967,18 +967,18 @@ private static int lookupName(final HBaseClient client,
...
+                                       final String kind,
+                                       final String name) {
     final UniqueId uid = new UniqueId(client, table, kind, (int) idwidth);
     try {
       final byte[] id = uid.getId(name);
       System.out.println(kind + ' ' + name + ": " + Arrays.toString(id));
       return 0;
     } catch (NoSuchUniqueName e) {
-      LOG.error(e.getMessage());
+      LOG.warn(e.getMessage());
```
Comment (ssuchter) [inline src/tools/UidManager.java]: Info not warn

File: src/tools/UidManager.java (hunk 1 above)
Comment (pepper-ankan) [inline src/tools/UidManager.java]: OK, sounds good.

File: src/tools/UidManager.java (hunk 2 above)
Comment (pepper-ankan) [inline src/tools/UidManager.java]: Sure.

**Comment by pepper-ankan** [comment]
Made the changes suggested by Sean.  Going to merge now.

**Pull request merged**